src/gbd_mapping_generator/data.py

```diff
@@ -322,12 +322,17 @@ def get_coverage_gap_list():
 
 def get_coverage_gap_data():
     out = _SPECIAL_COVERAGE_GAPS
-
     for c in gbd.get_coverage_gap_list():
-        metadata = get_coverage_gap_metadata(c)['exposure']
-        restrictions = tuple((k, v) for k, v in metadata['restrictions'].items())
-        levels = tuple((k, v) for k, v in metadata['levels'].items())
-        out.append((c, None, metadata['distribution'], restrictions, levels,
-                    metadata['affected_causes'], metadata['affected_risks']))
+        metadata = get_coverage_gap_metadata(c)
+        restrictions = tuple((k, v) for k, v in metadata['exposure']['restrictions'].items())
+        levels = tuple((k, v) for k, v in metadata['exposure']['levels'].items())
+
+        if metadata.get('relative_risk'):
+            affected_causes = metadata['relative_risk'].get('affected_causes')
```

Comment:
`get_coverage_gap_metadata` should guarantee that all the keys exists and fill the appropriate keys with `None` if no data is available.

Reply:
I did it because I didn't want to have something like `causes.None` or `causes.[]`which kept happening but let me think about it in a different way or maybe make a change in somewhere else to prevent that